[refactor] Remove unimplmented passes

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#26978 [refactor] Remove unimplmented passes**
* #26839 [refactor] Move some class/functions in test_jit.py to jit_utils.py

Summary:
We can add them later if there is a need.

Test Plan:
ci

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D17643009](https://our.internmc.facebook.com/intern/diff/D17643009)

jerryzh168 committed:
[refactor] Remove unimplmented passes

Summary:
We can add them later if there is a need.

Test Plan:
ci

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on torch/csrc/jit/passes/quantization.cpp:
```diff
@@ -621,14 +621,6 @@ script::Module InsertQuantDeQuant(
 }
 
 // PyBind APIs
-void PropagateQuantInfo(std::shared_ptr<Graph>& graph) {
```
The comment above can also be removed I think.

ZolotukhinM reviewed (approved):
👍

jerryzh168 committed:
Update on "[refactor] Remove unimplmented passes"


Summary:
We can add them later if there is a need.

Test Plan:
ci

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[refactor] Remove unimplmented passes"


Summary:
We can add them later if there is a need.

Test Plan:
ci

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]
